docs/index.rst

```diff
@@ -21,6 +21,12 @@ development and continuous integration ensure stable and reliable code. Pyfar
 packages are provided under the MIT open source license allowing unrestricted
 educational and commercial use.
 
+.. admonition:: Getting started
+
+      - If you are new to pyfar, we recommend the `Pyfar examples gallery`_, which works online.
+      - If you want to install a specific package, please refer to their documentation below.
+      - If you trouble with pyfar, please refer to the `Help section`_.
```

Comment:
Just noticed this

```suggestion
      - If you have trouble with pyfar, please refer to the `Help section`_.
```

Also, should there maybe be one short sentence linking to the contribute page?

Reply:
Thanks, I corrected the typo. I would not link to the contributing page here, because this is most likely not what you want when you're getting starte :)